sql/python/sqlflow/db.py

```diff
@@ -0,0 +1,65 @@
+
+
+def connect(driver, database, user, password, host, port):
+    if driver == "mysql":
+        from mysql.connector import connect
+        return connect(user=user,
+                       passwd=password,
+                       database=database,
+                       host=host,
+                       port=port)
+    elif driver == "sqlite3":
+        from sqlite3 import connect
+        return connect(database)
+    elif driver == "hive":
+        from impala.dbapi import connect
+        return connect(user=user,
+                       passwd=password,
+                       database=database,
+                       host=host,
+                       port=port)
+
+    raise ValueError("unrecognized database driver: %s" % driver)
+
+
+def execute(driver, conn, statement):
+    cursor = conn.cursor()
+    cursor.execute(statement)
+
+    if driver == "hive":
+        field_names = None if cursor.description is None \
+            else [i[0][i[0].find('.') + 1:] for i in cursor.description]
+    else:
+        field_names = None if cursor.description is None \
+            else [i[0] for i in cursor.description]
+
+    field_columns = list(map(list, zip(*cursor.fetchall())))
+
+    return field_names, field_columns
+
+
+def insert_values(driver, conn, table_name, table_schema, values):
+    if driver == "mysql":
+        statement = '''insert into {} ({}) values()'''.format(
+            table_name,
+            ", ".join([n for n, t in table_schema]),
+            ", ".join(["%s"] * len(table_schema))
+        )
+    elif driver == "sqlite3":
+        statement = '''insert into {} ({}) values({})'''.format(
+            table_name,
+            ", ".join([n for n, t in table_schema]),
+            ", ".join(["?"] * len(table_schema))
+        )
+    elif driver == "hive":
+        statement = '''insert into table {} ({}) values()'''.format(
+            table_name,
+            ", ".join([n for n, t in table_schema]),
+            ", ".join(["%s"] * len(table_schema))
+        )
+    else:
+        raise ValueError("unrecognized database driver: %s" % driver)
+
+    cursor = conn.cursor()
+    cursor.executemany(statement, values)
```

Comment:
```
test_hive (sqlflow.db_test.TestDB) ... ok
test_mysql (sqlflow.db_test.TestDB) ... ERROR
test_sqlite3 (sqlflow.db_test.TestDB) ... ok
======================================================================
ERROR: test_mysql (sqlflow.db_test.TestDB)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/go/src/github.com/sql-machine-learning/sqlflow/sql/python/sqlflow/db_test.py", line 29, in test_mysql
    self._do_test(driver, conn)
  File "/go/src/github.com/sql-machine-learning/sqlflow/sql/python/sqlflow/db_test.py", line 44, in _do_test
    execute(driver, conn, self.drop_statement)
  File "/go/src/github.com/sql-machine-learning/sqlflow/sql/python/sqlflow/db.py", line 36, in execute
    field_columns = list(map(list, zip(*cursor.fetchall())))
  File "/miniconda/envs/sqlflow-dev/lib/python3.6/site-packages/mysql/connector/cursor_cext.py", line 489, in fetchall
    raise errors.InterfaceError("No result set to fetch from.")
mysql.connector.errors.InterfaceError: No result set to fetch from.
```

@uuleon 
CI failed on MySQL with `No result set to fetch from.`. Do we have to `coursor.commit()` after `executemany()`?

Reply:
I don't think so. From the error stack information, the result of `fetchall` is indeed None because of the query is `drop table ***`, I have added code to handle this situation.

Reply:
Great!

Reply:
The actual reason for this failure is the syntax of `fetchall` is different for `MySQL` and `Sqlite3` when there are no results from `execute`, `None` is returned from `Sqlite3` and `InterfaceError` is raised by `MySQL`.